grass7/raster/r.mapcalc.tiled/r.mapcalc.tiled.html

```diff
@@ -0,0 +1,45 @@
+<h2>DESCRIPTION</h2>
+
+<em>r.mapcalc.tiled</em> cuts a raster input map into tiles and runs 
+<a href="r.mapcalc.html">r.mapcalc</a> over these tiles before patching the
+result together into a single output raster map.
+
+<p>
+The user provides the map calculation <b>expression</b>. The output mapname
+is automatically extracted from that expression by extracting the string before 
+the '='. If the <b>expression</b> is more complex, the user can also provide the
+the output map name with the parameter <b>output</b>
+
+<p>
+Tiles can be defined with the <b>width</b>, <b>height</b> and 
+<b>overlap</b> parameters. If <b>processes</b> is higher than one, these tiles 
+will be processed in parallel.
+
+<p>
+The <b>mapset_prefix</b> parameter allows to make sure that the temporary
+mapsets created during the tiled processing have unique names. This is useful
```

Comment:
```suggestion
mapsets created during the tiles processing have unique names. This is useful
```

Reply:
It is about the processing which is tiled, not the processing of the tiles...